src/renderer/marketplace/bank-accounts/list/index.jsx

```diff
@@ -1,40 +1,53 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
 import { pricesSelectors } from 'common/prices';
 import { withStyles } from '@material-ui/core/styles';
 import { bankAccountsOperations, bankAccountsSelectors } from 'common/bank-accounts';
+/*
 import { Grid } from '@material-ui/core';
 import CircularProgress from '@material-ui/core/CircularProgress';
+*/
+import { BankingOffersPage } from './offers-page';
 
-const styles = theme => ({
-	loading: {
-		marginTop: '5em'
-	}
-});
+const styles = theme => ({});
+const MARKETPLACE_ROOT_PATH = '/main/marketplace-categories';
 
 class BankAccountsTable extends Component {
```

Comment:
I think we should add 'Container' suffix to all container elements to easily distinguish between pure ui elements and containers

Agree?

Reply:
Agreed.